src/containers/Application/ApplicationHome.tsx

```diff
@@ -49,19 +49,19 @@ const ApplicationHome: React.FC<ApplicationProps> = ({ structure, template }) =>
   } = fullStructure
 
   return (
-    <ApplicationHeader template={template} currentUser={currentUser}>
+    <>
       <ChangesRequestedTitle status={current?.status} isChangeRequest={isChangeRequest} />
-      <ApplicationHomeWrapper startMessage={template.startMessage}>
+      <ApplicationHomeWrapper startMessage={template.startMessage} name={template.name}>
         <ApplicationSections fullStructure={structure} />
+        {current?.status === ApplicationStatus.Draft && !firstStrictInvalidPage && (
+          <Segment basic className="padding-zero" textAlign="right">
+            <Button as={Link} color="blue" onClick={handleSummaryClicked}>
+              {strings.BUTTON_SUMMARY}
+            </Button>
+          </Segment>
```

Comment:
For consistency it might be better to pass this button the same way you did for ApplicationCreate, BTW this button isn't included in the original designs, I just though that, if all sections are completed and valid, we should allow the user to go straight to the `ApplicationSummary`  from `ApplicationHome`, peek how it display now:
![Screen Shot 2021-05-04 at 10 05 11 AM](https://user-images.githubusercontent.com/16461988/116939321-42d6be00-acc0-11eb-93b3-31d7f48d4d57.png)

Reply:
The only reason I did it like that in ApplicationCreate was because the Button was conditional, whereas this one is always present on ApplicationHome (isn't it?), so should we just leave it as is? (I think that seems better than polluting ApplicationSections with an additional button, as it's already pretty full, wouldn't you say?)

Reply:
> The only reason I did it like that in ApplicationCreate was because the Button was conditional, whereas this one is always present on ApplicationHome (isn't it?)

No - only when ALL responses are completed and valid. Otherwise it would show a `Resume` button like this:
![Screen Shot 2021-05-04 at 12 17 28 PM](https://user-images.githubusercontent.com/16461988/116948136-ba154d80-acd2-11eb-9ee1-23deb6bb9418.png)

Reply:
Ah, okay, I'll update this shortly.

Reply:
Cool, this makes more sense now, thanks for picking it up. deb0e3d8